test/bigtable/index.js

```diff
@@ -219,6 +222,44 @@ describe('Bigtable', function() {
       bigtable.createTable(TABLE_ID, options, assert.ifError);
     });
 
+    it('should set the column families', function(done) {
```

Comment:
Can you create these tests:

- describe('options.families', ...
  - it should accept a string family name
  - it should accept a string garbage collection rule
  - it should accept an object garbage collection rule

The reason being, if someone makes a change to your test that currently combines these, they might accidentally remove one of the checks. Having individual tests that can fail will reduce that chance.